keps/sig-instrumentation/1668-trace-popagating/kep.yaml

```diff
@@ -0,0 +1,46 @@
+title: trace information popagation
+kep-number: 1668
+authors:
+ - "@hase1128"
+ - "@KobayashiD27"
+ - "@fenggw-fnst"
+ - "@zhijianli88"
+ - "@Hellcatlk"
+owning-sig: sig-instrumentation
+participating-sigs:
```

Comment:
Can you please pull in SIG api-machinery? I think we may also need a thumbs up from Architecture but let's start with them.

Reply:
Thanks. Done.